src/pymatgen/io/multiwfn.py

```diff
@@ -454,7 +454,7 @@ def sort_cps_by_distance(
 
 def process_multiwfn_qtaim(
     molecule: Molecule,
-    file: Union[str, Path],
+    file: str | Path,
```

Comment:
Please feel free to ping me if you hate fighting with `mypy` :)

We now have a type alias for "path like" stuff like this:
```python
from pymatgen.util.typing import PathLike
```